bin/impl/install.sh

```diff
@@ -15,6 +15,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# shellcheck source=bin/impl/util.sh
```

Comment:
What is the purpose of this comment? does it help shellcheck find the file being sourced?

Reply:
Yes, exactly.